docs/org/reference/tooling/emacs.org

```diff
@@ -3,11 +3,17 @@
 There is an Emacs mode available for Juvix. Currently, it
 supports syntax highlighting for well-scoped modules.
 
+To get started, clone the Juvix Emacs mode repository:
+
+#+begin_src sh
+git clone https://github.com/anoma/emacs-juvix.git
```

Comment:
even though this link seems to still, work, we should probably have `https://github.com/anoma/juvix-mode`